extensions/wikia/FacebookClient/scripts/FacebookClient.facebookTags.js

```diff
@@ -12,7 +12,9 @@ mw.hook('wikipage.content').add(function ($content) {
 
 	// Checks for XFBML tags or Facebook Page Plugin tags
 	function facebookTagsOnPage() {
-		var numOfFacebookTags = $content.find('[data-type="xfbml-tag"], .fb-page').length;
+        // All fb social plugins have a class which starts with 'fb-'
```

Comment:
Bah! My IDE put in spaces instead of tabs! I'll fix that :)
![i should know better](https://camo.githubusercontent.com/96d51190854058fe8f6953dd4f1a50047ef47bd9/687474703a2f2f6576616465666c6f772e636f6d2f77702d636f6e74656e742f75706c6f6164732f323031312f30332f54616273537061636573426f74682e706e67)